packages/node/src/store.ts

```diff
@@ -45,13 +45,10 @@ export class Store {
       `${this.storeKeyPrefix}/${DB_NAMESPACE_CHANNEL}`
     )) as { [multisigAddress: string]: StateChannelJSON };
 
-    if (!channelsJSON) {
-      console.log("No channels exist yet");
-    } else {
-      for (const entry of Object.entries(channelsJSON)) {
-        channels[entry[0]] = StateChannel.fromJson(entry[1]);
-      }
+    for (const entry of Object.entries(channelsJSON || {})) {
+      channels[entry[0]] = StateChannel.fromJson(entry[1]);
```

Comment:
@snario We should have some casing consistency: variable is named `channelsJSON` (all caps), type is named `StateChannelJSON` (also all caps) but the function is called `fromJson` (word-case).

Reply:
I think perhaps we should do this in a separate PR